### ISSUE TITLE
[Cata] Lua errors when mousing over certain objects in Scholomance

### ISSUE THREAD

**Issue by entroscopy** [Questie/Questie#6348]
## Bug description

Mousing over the torch lever near Jandice Barov, and the little hidden chest it opens, throws lua errors.

## Screenshots

Offending objects:
![image](https://github.com/user-attachments/assets/41bae05b-f7c1-4417-91e3-3613789ae39f)
![image](https://github.com/user-attachments/assets/becbfe88-b165-4e83-bacd-4ccad4563b72)

## Questie & WoW version

Questie 10.3.10
Wow 4.4.0.55639

```
Date: 2024-08-03 08:17:58
ID: 1
Error occured in: Global
Count: 12
Message: Interface/AddOns/Questie/Database/Zones/zoneDB.lua line 119:
   No AreaId found for UiMapId: 307:ScholomanceOLD
Debug:
   [string "=[C]"]: error()
   [string "@Interface/AddOns/Questie/Database/Zones/zoneDB.lua"]:119:
      Interface/AddOns/Questie/Database/Zones/zoneDB.lua:77
   [string "=(tail call)"]: ?
   [string "@Interface/AddOns/Questie/Modules/Tooltips/Tooltip.lua"]:210: GetTooltip()
   [string "@Interface/AddOns/Questie/Modules/Tooltips/TooltipHandler.lua"]:101: AddObjectDataToTooltip()
   [string "@Interface/AddOns/Questie/Modules/Tooltips/Tooltip.lua"]:437:
      ...nterface/AddOns/Questie/Modules/Tooltips/Tooltip.lua:420
   [string "=[C]"]: ?
Locals:
(*temporary) = "No AreaId found for UiMapId: 307:ScholomanceOLD"
```
```
Date: 2024-08-03 08:15:36
ID: 2
Error occured in: Global
Count: 462
Message: Interface/AddOns/Questie/Database/Zones/zoneDB.lua line 119:
   No AreaId found for UiMapId: 308:ScholomanceOLD
Debug:
   [string "=[C]"]: error()
   [string "@Interface/AddOns/Questie/Database/Zones/zoneDB.lua"]:119:
      Interface/AddOns/Questie/Database/Zones/zoneDB.lua:77
   [string "=(tail call)"]: ?
   [string "@Interface/AddOns/Questie/Modules/Tooltips/Tooltip.lua"]:210: GetTooltip()
   [string "@Interface/AddOns/Questie/Modules/Tooltips/TooltipHandler.lua"]:101: AddObjectDataToTooltip()
   [string "@Interface/AddOns/Questie/Modules/Tooltips/Tooltip.lua"]:437:
      ...nterface/AddOns/Questie/Modules/Tooltips/Tooltip.lua:420
   [string "=[C]"]: ?
Locals:
(*temporary) = "No AreaId found for UiMapId: 308:ScholomanceOLD"
```

**Comment by BreakBB** [comment]
Thanks @entroscopy great report!

We'll look into this 👍🏻

**Comment by BreakBB** [comment]
@Cabro this is something for you to look into, right?

**Comment by BreakBB** [comment]
Fixed in 8b5d793d42c88721e4329ee6db3184d894ae011d and released with v10.3.14
